quint/src/graphics.ts

```diff
@@ -123,6 +121,99 @@ export function prettyQuintEx(ex: QuintEx): Doc {
   }
 }
 
+export function prettyQuintDef(
+  def: QuintDef,
+  includeBody: boolean = true,
+  type: TypeScheme | undefined = undefined
+): Doc {
+  const typeAnnotation = isAnnotatedDef(def)
+    ? [text(': '), prettyQuintType(def.typeAnnotation)]
+    : type
+    ? [text(': '), prettyTypeScheme(type)]
+    : []
```

Comment:
Could you add parentheses here?

Reply:
same as https://github.com/informalsystems/quint/pull/900#discussion_r1200396964